toolchain/sem_ir/formatter.cpp

```diff
@@ -230,19 +226,26 @@ class FormatterImpl {
     Indent(-2);
   }
 
-  // Formats a top-level scope, particularly Constants and ImportRefs.
-  auto FormatScope(InstNamer::ScopeId scope_id, llvm::ArrayRef<InstId> block,
-                   bool tentative = false) -> void {
+  // Formats a top-level scope, and any of the instructions in that scope that
+  // are used.
+  auto FormatScopeIfUsed(InstNamer::ScopeId scope_id,
```

Comment:
FWIW, you might consider "FormatTentativeScope" or similar, to match that terminology. "IfUsed" makes me kind of wonder if it's saying the scope itself might not be used, when it's actually the instructions contained by the scope that are conditional.